client/packages/invoices/src/utils.ts

```diff
@@ -313,6 +313,8 @@ export const isA = {
 export const get = {
   stockLineSubtotal: (line: DraftStockOutLine) =>
     line.numberOfPacks * (line.stockLine?.sellPricePerPack ?? 0),
+  invoiceLineSubtotal: (line: DraftStockOutLine) =>
```

Comment:
I guess this would only be kept in downstream branch if used in UI display (if needed)

Reply:
https://github.com/msupply-foundation/open-msupply/issues/4763